index.js

```diff
@@ -53,17 +53,20 @@ Allure.prototype.endCase = function(status, err, timestamp) {
 Allure.prototype.startStep = function(stepName, timestamp) {
     var step = new Step(stepName, timestamp),
         suite = this.getCurrentSuite();
-    step.parent = suite.currentStep;
-    step.parent.addStep(step);
-    suite.currentStep = step;
+    if (suite.currentStep) {
+        step.parent = suite.currentStep;
+        step.parent.addStep(step);
+        suite.currentStep = step;
+    }
```

Comment:
If we could not find the current step, we need to write a warning into console, like you are doing below in the `addAttachment` method.

Reply:
I thought about it, but removed this log on purpose – to prevent flooding the console in (mine) common case of beforeAll

But you are right, and I can add some silent mode if I really need it someday)